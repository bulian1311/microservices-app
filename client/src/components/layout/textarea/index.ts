import { TextArea } from "./textarea.styles";

export default TextArea;
