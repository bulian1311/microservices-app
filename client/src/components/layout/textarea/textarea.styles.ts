import styled from "styled-components";

export const TextArea = styled.textarea`
  width: 100%;
  height: 100%;
  background: var(--bg-color);
  box-shadow: inset 3px 3px 10px #c6c6c6, inset -3px -3px 10px #ffffff;
  border-radius: 4px;
  font-weight: 300;
  font-size: 1.3125rem;
  line-height: 1.190em;
  color: inherit;
  font-family: inherit;
  padding: 0.952em;
  ::placeholder {
    color: var(--text-color-light);
  }
`;
