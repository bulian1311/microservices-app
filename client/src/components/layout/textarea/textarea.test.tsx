import * as React from "react";
import { render, RenderResult } from "@testing-library/react";
import { TextArea } from "./textarea.styles";

let documentBody: RenderResult;

beforeEach(() => {
  documentBody = render(<TextArea />);
});

test("TextArea matces snapshot.", () => {
  const { baseElement } = documentBody;
  expect(baseElement).toMatchSnapshot();
});
