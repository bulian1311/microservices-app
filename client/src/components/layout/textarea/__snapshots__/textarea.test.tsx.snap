// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextArea matces snapshot. 1`] = `
<body>
  <div>
    <textarea
      class="sc-bdfBwQ jUhbqp"
    />
  </div>
</body>
`;
